pkgs/development/python-modules/pysol_cards/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, python3Packages }:
+
+buildPythonPackage rec {
+  pname = "pysol_cards";
+  version = "0.14.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-wI3oV1d7w+7JcMOt08RbNlMWzChErNYIO7Vuox1A6vA=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ six random2 ];
```

Comment:
Please move `six` and `random2` to the top arguments, `python3Packages` should not be used in python modules (applications are fine).

Reply:
Done.